parachain/pallets/ethereum-beacon-client/src/merkleization.rs

```diff
@@ -16,7 +18,33 @@ pub enum MerkleizationError {
     InvalidLength,
 }
 
-pub fn get_ssz_beacon_block_body(body: Body) -> Result<SSZBeaconBlockBody, MerkleizationError> {
+pub fn get_ssz_beacon_block_body<
+	FeeRecipientSize: Get<u32>, 
```

Comment:
Similar comment as above, could we make a type alias for `Body`?

Reply:
I couldn't get an alias to work for this module, because it doesn't specify the T::Config implementation of the generic here - it's all different types here. Do you know how I can do the alias outside of the pallet? @vgeddes maybe you also know? I tried a few things but couldn't get it to work. 🤔